CL/cl_ext.h

```diff
@@ -603,7 +623,7 @@ clRemapCommandBufferKHR(
     "cl_khr_command_buffer_mutable_dispatch"
 
 
-#define CL_KHR_COMMAND_BUFFER_MUTABLE_DISPATCH_EXTENSION_VERSION CL_MAKE_VERSION(0, 9, 1)
+#define CL_KHR_COMMAND_BUFFER_MUTABLE_DISPATCH_EXTENSION_VERSION CL_MAKE_VERSION(0, 9, 3)
```

Comment:
https://github.com/KhronosGroup/OpenCL-Headers/pull/245 needs to merge first, otherwise we bump to patch versions